VehicleCode/Input/AnalogSensor.cpp

```diff
@@ -12,18 +12,16 @@
  * Initializes an AnalogSensor object with specified parameters.
  *
  * @param id (int) The unique identifier for the sensor.
- * @param freq (int) The update delay of the sensor in milliseconds.
+ * @param waitTime (int) The update delay of the sensor in milliseconds.
  * @param inPins (int) The input pin id for the analog sensor.
  */
-AnalogSensor::AnalogSensor(int id, int waitTime, int inPin, int bias, int max, int dataLength) {
+AnalogSensor::AnalogSensor(int id, int waitTime, int inPin) {
```

Comment:
Why don't we rescale in here AnalogSensor instead of in AVA again?

Reply:
Preserve data resolution for analysis. Instead of 0-100% we get the full range saved to SD if needed. Leaves the option open for further more specific analysis if we want.